docs/assets/commonMarkdown/sharedDataUse.md

```diff
@@ -0,0 +1,126 @@
+
+```SQL
+CREATE STORAGE VOLUME def_volume
+TYPE = S3
+LOCATIONS = ("s3://defaultbucket/test/")
+PROPERTIES
+(
+    "enabled" = "true",
+    "aws.s3.region" = "us-west-2",
+    "aws.s3.endpoint" = "https://s3.us-west-2.amazonaws.com",
+    "aws.s3.use_aws_sdk_default_behavior" = "false",
+    "aws.s3.use_instance_profile" = "false",
+    "aws.s3.access_key" = "xxxxxxxxxx",
+    "aws.s3.secret_key" = "yyyyyyyyyy"
+);
+
+SET def_volume AS DEFAULT STORAGE VOLUME;
+```
+
+For more information on how to create a storage volume for other object storages and set the default storage volume, see [CREATE STORAGE VOLUME](../sql-reference/sql-statements/Administration/CREATE%20STORAGE%20VOLUME.md) and [SET DEFAULT STORAGE VOLUME](../sql-reference/sql-statements/Administration/SET%20DEFAULT%20STORAGE%20VOLUME.md).
+
+### Create a database and a cloud-native table
+
+After you create a default storage volume, you can then create a database and a cloud-native table using this storage volume.
+
+Currently, shared-data StarRocks clusters support the following table types:
+
+- Duplicate Key table
+- Aggregate table
+- Unique Key table
+- Primary Key table (Currently, the primary key persistent index is not supported.)
+
+The following example creates a database `cloud_db` and a table `detail_demo` based on Duplicate Key table type, enables the local disk cache, sets the hot data validity duration to one month, and disables asynchronous data ingestion into object storage:
+
+```SQL
+CREATE DATABASE cloud_db;
+USE cloud_db;
+CREATE TABLE IF NOT EXISTS detail_demo (
+    recruit_date  DATE           NOT NULL COMMENT "YYYY-MM-DD",
+    region_num    TINYINT        COMMENT "range [-128, 127]",
+    num_plate     SMALLINT       COMMENT "range [-32768, 32767] ",
+    tel           INT            COMMENT "range [-2147483648, 2147483647]",
+    id            BIGINT         COMMENT "range [-2^63 + 1 ~ 2^63 - 1]",
+    password      LARGEINT       COMMENT "range [-2^127 + 1 ~ 2^127 - 1]",
+    name          CHAR(20)       NOT NULL COMMENT "range char(m),m in (1-255) ",
+    profile       VARCHAR(500)   NOT NULL COMMENT "upper limit value 65533 bytes",
+    ispass        BOOLEAN        COMMENT "true/false")
+DUPLICATE KEY(recruit_date, region_num)
+DISTRIBUTED BY HASH(recruit_date, region_num)
+PROPERTIES (
+    "storage_volume" = "def_volume",
+    "datacache.enable" = "true",
+    "datacache.partition_duration" = "1 MONTH",
+    "enable_async_write_back" = "false"
+);
+```
+
+> **NOTE**
+>
+> The default storage volume is used when you create a database or a cloud-native table in a shared-data StarRocks cluster if no storage volume is specified.
+
+In addition to the regular table `PROPERTIES`, you need to specify the following `PROPERTIES` when creating a table for shared-data StarRocks cluster:
+
+#### datacache.enable
```

Comment:
one thing that could confusing the user is, we have a property rename from v3.0 to v3.1, so these properties are only for v3.1 and onwards, the property name is different in v3.0, may worth notice this.

Reply:
Thanks.